plugins/bc-favorite/tests/TestCase/BcFavoritePluginTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace BcFavorite\Test\TestCase;
+
+use BaserCore\TestSuite\BcTestCase;
+use BaserCore\Utility\BcUtil;
+use BcFavorite\Service\FavoritesServiceInterface;
+use Cake\Core\Container;
+use Cake\Core\Plugin;
+
+class BcFavoritePluginTest extends BcTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        BcUtil::includePluginClass('BcFavorite');
+        $plugins = Plugin::getCollection();
+        $this->Plugin = $plugins->create('BcFavorite');
+        $plugins->add($this->Plugin);
```

Comment:
@thangnnmd  dòng 16 tới 19 a thay = cái này có được ko a 
$this->BcFavoritePlugin = new BcFavoritePlugin(['name' => 'BcFavorite']);

Reply:
@HungDV2022 anh có sửa lại rồi nhé